components/camel-infinispan/camel-infinispan/pom.xml

```diff
@@ -64,6 +64,16 @@
             <artifactId>infinispan-remote-query-client</artifactId>
             <version>${infinispan-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan.protostream</groupId>
+            <artifactId>protostream</artifactId>
+            <version>${protostream-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan.protostream</groupId>
+            <artifactId>protostream-types</artifactId>
+            <version>${protostream-version}</version>
+        </dependency>
```

Comment:
realizing I left direct dependecy and dependency mangement, will remove one of them